src/extension.ts

```diff
@@ -269,6 +272,6 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 
 export async function deactivate(): Promise<void> {
   if (lsClient) {
-    await lsClient.stop();
+    await stopServer(lsClient);
   }
 }
```

Comment:
Also, this seems like a bugfix because otherwise the listeners wouldn't be disposed.